projects/demo/src/modules/components/avatar/examples/6/index.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, Inject} from '@angular/core';
+import {DomSanitizer} from '@angular/platform-browser';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+
+@Component({
+    selector: 'tui-avatar-example-6',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+})
+export class TuiAvatarExample6 {
+    readonly waterplea = 'https://avatars.githubusercontent.com/u/11832552?v=4';
+    readonly wrongUrl = 'https://google.com/images/not-found.png';
+
+    readonly fallback = this.sanitizer.bypassSecurityTrustHtml(
```

Comment:
Avatar works with tuiIcon icons, let's not add this and just use a named icon as a fallback.